src/creator.coffee

```diff
@@ -234,6 +239,13 @@ Hangman.controller 'HangmanCreatorCtrl', ['$timeout', '$scope', '$sanitize', 'Re
 		,
 		400
 
+	$scope.openQuestionBankDialog = ->
+		$scope.questionBankDialog = true
+		$scope.enableQuestionBank = false
+
+	$scope.closeQuestionBankDialog = ->
```

Comment:
You might try (if you haven't already) simply setting `questionBankDialog = false` as an expression in the associated `ng-click` value in the creator DOM, since creating a function to do that is redundant. Granted, I know angularjs can (sometimes) have issues with those expressions.